version/version_lib.go

```diff
@@ -0,0 +1,16 @@
+// +build !autogen
+
+// Package version is auto-generated at build-time
+package version
+
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GITCOMMIT string = "library-import"
```

Comment:
@vdemeester I was under the impression these were new. I suppose we can keep them around and update them in another PR. I'm surprised `golint` let this pass.